packages/fxa-auth-server/docs/api.md

```diff
@@ -3008,6 +3020,32 @@ Returns SMS status for the current user.
   is in the specified country.
   <!--end-query-param-get-smsstatus-country-->
 
+### Subscriptions
+
+#### GET /oauth/subscriptions/plans
+
+:lock: authenticated with OAuth bearer token
+Returns a list of available subscription plans.
+
+#### GET /oauth/subscriptions/active
+
+:lock: authenticated with OAuth bearer token
+Returns a list of active subscriptions for the user.
+
+#### POST /oauth/subscriptions/active
+
+:lock: authenticated with OAuth bearer token
+Subscribe the user to a plan using a payment token.
```

Comment:
docs nit: "payment token" is not defined anywhere in this document, is there some other document that it could link out to with more details?

Reply:
It's an opaque token representing a payment method and consent to pay. Defined by [subhub](https://github.com/mozilla/subhub), which in turn takes it from [Stripe.js](https://stripe.com/docs/stripe-js/reference#stripe-create-token). But, subhub is a private project and we're trying to stay more agnostic to the underlying payment vendor. TL;DR: No great docs to link to about this